src/main/java/com/exasol/dbbuilder/dialects/postgres/PostgreSqlSchema.java

```diff
@@ -0,0 +1,33 @@
+package com.exasol.dbbuilder.dialects.postgres;
+
+import com.exasol.db.Identifier;
+import com.exasol.dbbuilder.dialects.*;
+
+/**
+ * PostgreSQL {@link Schema}.
+ */
+public class PostgreSqlSchema extends AbstractSchema {
+    private final PostgreSqlImmediateDatabaseObjectWriter writer;
+
+    /**
+     * Create a new instance of {@link PostgreSqlSchema}.
+     * 
+     * @param writer object writer
+     * @param name   name of the schema
+     */
+    public PostgreSqlSchema(final PostgreSqlImmediateDatabaseObjectWriter writer, final String name) {
+        super(PostgreSqlIdentifier.of(name));
```

Comment:
same here, move this to an argument in the constructor instead of having the hardcoded dependency, as you are doing it already in `PostgreSqlUser` for example.

Reply:
I understand we want to have the same interface for all dialects. Please create a ticket to change them all and unify them in the future